enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = "0.31.4"
+__version__ = "0.31.5"
```

Comment:
Just a heads up that we're now up to [v0.32.0](https://github.com/edx/edx-enterprise/releases/tag/0.32.0) for this project